src/handlers/request-handlers/root-request-handler.ts

```diff
@@ -26,14 +26,13 @@ export const rootRequestHandler = (request: Request, response: Response, next: N
       software: packageJson.repository.url,
       version: packageJson.version,
       limitation: {
-            max_message_length: settings.network.maxPayloadSize,
+            max_payload_size: settings.network.maxPayloadSize,
             max_subscriptions: settings.limits.client.subscription.maxSubscriptions,
             max_filters: settings.limits.client.subscription.maxFilters,
-            max_limit: 5000,
             max_subid_length: 256,
             min_prefix: 4,
             max_event_tags: 2500,
-            max_content_length: 102400,
+            max_content_length: settings.limits.event.content,
```

Comment:
settings.limits.event.content is either ContentLimits or an array of ContentLimits.

 https://github.com/Cameri/nostream/blob/main/src/%40types/settings.ts#L58

Reply:
Won't this work in either situation? Won't both be serialized accordingly?  

As is, it's a hardcoded constant that's not representative of reality.  It was confusing to see my relay report this number, and no matter what I changed in settings, the information didn't change.

Reply:
Oops, docs were missed on some update.

Reply:
Fixed in a separate PR